CONTRIBUTING.md

```diff
@@ -33,16 +39,28 @@ $ git clone git@github.com:<user>/kubebuilder.git $GOPATH/src/sigs.k8s.io/kubebu
 1. Build the project by using the command `make build` 
 1. Run the command `make install` to create a bin with the source code 
 
-## How to test kubebuilder locally
+**NOTE** In order to check the local environment run `make go-test`.
+
+## What to do before submitting a pull request 
 
-1. Run the tests by using the command `make test`. It will execute unit tests. 
-1. Run the script `make generate` to update/generate the mock data used in the e2e test in `$GOPATH/src/sigs.k8s.io/kubebuilder/testdata/`
+1. Run the script `make generate` to update/generate the mock data used in the e2e test in `$GOPATH/src/sigs.k8s.io/kubebuilder/testdata/` 
+2. Run `make lint` to check your code.
```

Comment:
Leave all as ones as in the rest of the document, markdown will properly place the right number.

```suggestion
1. Run `make lint` to check your code.
```

Reply:
Move it for the NOTE since it is not really mandatory to.